src/analysis/retail/shaman/restoration/modules/dragonflight/Tier30.tsx

```diff
@@ -0,0 +1,240 @@
+        event.timestamp,
+        CAST_BUFFER_MS,
+      )
+    ) {
+      return;
+    }
+    const healIncrease = this.current4pcStacks * TIER_4PC_SWELLING_RAIN_HSHW_PER_STACK;
+    this.swellingRainHealingSurge += calculateEffectiveHealing(event, healIncrease);
+  }
+
+  handleChainHeal(event: CastEvent) {
+    if (
+      !this.selectedCombatant.hasBuff(
+        ITEMS.T30_SWELLING_RAIN_BUFF.id,
+        event.timestamp,
+        CAST_BUFFER_MS,
+      )
+    ) {
+      return;
+    }
+    const healIncrease = this.current4pcStacks * TIER_4PC_SWELLING_RAIN_CHAINHEAL_PER_STACK;
+    const orderedChainHeal = this.chainHealNormalizer.normalizeChainHealOrder(event);
+    debug && console.log('chain heal: ', orderedChainHeal, healIncrease);
+    if (orderedChainHeal.length > 0) {
+      this.swellingRainChainheal += this._tallyHealingIncrease(orderedChainHeal, healIncrease);
+      debug && console.log(this.swellingRainChainheal);
+    }
+  }
+
+  private _tallyHealingIncrease(events: HealEvent[], healIncrease: number): number {
+    if (events.length > 0) {
+      return events.reduce(
+        (amount, event) => amount + calculateEffectiveHealing(event, healIncrease),
+        0,
+      );
+    }
+    return 0;
+  }
+
+  statistic() {
+    return (
+      <Statistic
+        size="flexible"
+        position={STATISTIC_ORDER.OPTIONAL(0)}
+        category={STATISTIC_CATEGORY.ITEMS}
+        tooltip={
+          <>
+            <ul>
+              <li>
+                <SpellLink id={ITEMS.T30_TIDEWATERS_HEAL.id} />:{' '}
+                <strong>{formatNumber(this.tidewatersHealing)}</strong> (
+                {this.twoPieceOverhealPercent}% Overheal)
+              </li>
+              {this.has4pc && (
+                <>
+                  <li>
+                    <SpellLink id={ITEMS.T30_RAINSTORM_BUFF.id} />:{' '}
+                    <strong>{formatNumber(this.rainstormHealing)}</strong>
+                  </li>
+                  <SpellLink id={ITEMS.T30_SWELLING_RAIN_BUFF.id} />:
+                  <li>
+                    <SpellLink id={talents.HEALING_WAVE_TALENT.id} />:{' '}
+                    <strong>{formatNumber(this.swellingRainHealingWave)}</strong>
+                  </li>
+                  <li>
+                    <SpellLink id={talents.CHAIN_HEAL_TALENT.id} />:{' '}
+                    <strong>{formatNumber(this.swellingRainChainheal)}</strong>
+                  </li>
+                  <li>
+                    <SpellLink id={SPELLS.HEALING_SURGE.id} />:{' '}
+                    <strong>{formatNumber(this.swellingRainHealingSurge)}</strong>
+                  </li>
+                </>
+              )}
+            </ul>
+          </>
+        }
+      >
+        <div className="pad boring-text">
+          <label>
+            <ItemSetLink id={SHAMAN_T30_ID}>
+              <>
+                Runes of the Cinderwolf
+                <br />
+                (Aberrus Tier)
```

Comment:
```suggestion
                (T30 Tier Set)
```
to be consistent with our other modules